lib/reporter.dart

```diff
@@ -18,8 +20,8 @@ class Reporter {
     @required String rootFolderName,
     @required String groupName,
   }) async {
-    final ozzieFiles = await _getOzzieFiles(rootFolderName);
-    final imageGallery = _buildImageGallery(ozzieFiles);
+    final ozzieFiles = await _getOzzieReports(rootFolderName);
+    final imageGallery = _buildOzzieReport(ozzieFiles);
```

Comment:
i'm confused haha what's the difference between `_getOzzieReports` and `_buildOzzieReport`. Can we rename these to be a bit more descriptive?